hypothesis.py

```diff
@@ -101,35 +110,45 @@ def fraud_metrics(sample_data: pd.DataFrame,
     false_positive = 0
     false_negative = 0
     for group_key, isotope_values in sample_data:
-        if isotope_values.size <= 1:
-            continue
-        hypothesis_test = sample_ttest(group_key[1], group_key[2], isotope_values, means_isoscape,
-                                       variances_isoscape, sample_size_per_location, p_value_target)
-
-        if not group_key[3]:
-            if hypothesis_test.p_value >= p_value_target:
-                true_negative += 1
-            else:
-                false_positive += 1
+      if isotope_values.shape[0] <= 1:
+        continue
+
+      p_values = []
+      for i, isotope_column_name in enumerate(isotope_column_names):
+        hypothesis_test = sample_ttest(group_key[1],
+                                       group_key[2],
+                                       isotope_values[isotope_column_name],
+                                       means_isoscapes[i],
+                                       variances_isoscapes[i],
+                                       sample_size_per_location,
+                                       p_value_target)
+        p_values.append(hypothesis_test.p_value)
+      combined_p_value = np.array(p_values).prod()
```

Comment:
Could we consider Pearson's method for combining p-values as discussed in our doc: https://docs.google.com/document/d/1sEMXTH3cQsEIuYcWJtWShiep3QUrsGh7c19hs4HEH38/edit#heading=h.hjjnxxv0m23t

Reply:
This is implementing what we had already (and decided to do at first). Let's implement this in a subsequent PR so we can experiment and validate. The former has already been used before.